frontend/src/pages/Nodes/NodesTable.js

```diff
@@ -1,78 +1,58 @@
 import React from "react";
-import Table from "@material-ui/core/Table";
-import TableHead from "@material-ui/core/TableHead";
-import TableRow from "@material-ui/core/TableRow";
-import TableCell from "@material-ui/core/TableCell";
-import TableBody from "@material-ui/core/TableBody";
-import Typography from "@material-ui/core/Typography";
-import Paper from "@material-ui/core/Paper";
 import { withStyles } from "@material-ui/core/styles";
-import blueGrey from "@material-ui/core/colors/blueGrey";
-import strings from "../../localizeStrings";
+import AppBar from "@material-ui/core/AppBar";
+import Tab from "@material-ui/core/Tab";
+import Tabs from "@material-ui/core/Tabs";
+import ApprovedNodesTable from "./ApprovedNodesTable";
+import DeclinedNodesTable from "./DeclinedNodesTable";
 
 const styles = {
-  paper: {
-    marginTop: "40px"
-  },
-  title: {
+  container: {
     width: "100%",
     display: "flex",
-    justifyContent: "center",
-    height: "50px",
-    alignItems: "center",
-    backgroundColor: blueGrey[50]
+    justifyContent: "center"
+  },
+  customWidth: {
+    width: "100%",
+    marginTop: "40px"
   }
 };
 
-const groupBy = nodes => {
-  return nodes.reduce((acc, node) => {
-    if (node.currentAccess.accessType !== "none") {
-      const key = node.address.organization;
-      const index = acc.findIndex(x => x.organization === key);
-      const permissionsString = `${node.currentAccess.accessType} (${node.isConnected ? strings.status.connection : strings.status.not_connected})`;
-      if (index === -1) {
-        acc.push({ organization: key, count: 1, permissions: permissionsString});
-      } else {
-        acc[index].count += 1;
-        acc[index].permissions = acc[index].permissions.concat(`, ${permissionsString}`);
-      }
-    }
-    return acc;
-  }, []);
+
+const renderTab = props => {
+  const {
+    tabIndex
+  } = props;
+  switch (tabIndex) {
+    case 0:
+      return (<ApprovedNodesTable {...props} />);
+    case 1:
+      return (<DeclinedNodesTable {...props} />);
+    default:
+      break;
+  }
+  return null;
 };
 
-const NodesTable = ({ nodes, classes }) => {
-  const groupedNodes = groupBy(nodes);
+const NodesTable = props => {
+  const { classes, tabIndex, setTabIndex } = props;
```

Comment:
tabIndex and setTabIndex are state variables for NodesTabel component. Use React.useState() instead of redux-state